### ISSUE TITLE
Is there a way to retrieve a list of members from a get_group_info

### ISSUE THREAD

**Issue by SenExus** [kevinzg/facebook-scraper#696]
Is there currently a way to retrieve a list of members using get_group_info? I did not see something of the sort in the documentation, so I was wondering if there was such a thing. Thanks.

**Comment by neon-ninja** [comment]
Yes, all non-admin members that can be extracted are listed in the result, under the `other_members` key

**Comment by lorellav** [comment]
Hi, is it possible to still only get 116 group members or it is now updated to get all members? I tried `get_group_info` but it only returns 116 members. Thank you!

**Comment by neon-ninja** [comment]
It's still limited (by Facebook) to 116

**Comment by lorellav** [comment]
Oh ok thank you!
